templates/admin/emails/list.tmpl

```diff
@@ -33,9 +33,9 @@
 			<table class="ui very basic striped table">
 				<thead>
 					<tr>
-						<th>{{.i18n.Tr "admin.users.name"}}</th>
+						<th data-sortt-normal="username" data-sortt-reverse="reverseusername">{{.i18n.Tr "admin.users.name"}}</th>
```

Comment:
I haven't read all the code, but could this maybe be generalized to `data-sortt-by` and `data-sortt-direction` (desc, asc)?

Reply:
I renamed the data attributes to asc and desc but since the oposit name of a sort option can't be created in general so we have to set the name of each asc and desc manualy.

@silverwind look at: https://github.com/go-gitea/gitea/pull/7980/files#diff-b7a30121512853ff07987fc7220e1503R19